core/dcschema.mcf

```diff
@@ -2858,3 +2858,24 @@ Node: dcid:Portion
 typeOf: dcs:UnitOfMeasure
 name: "Portion"
 description: "A unit of measure used to measure serving of food."
+
+Node: dcid:exchangeRate
+typeOf: schema:Property
+name: "exchangeRate"
+description: "The exchange rate of a currency compared to another currency."
+domainIncludes: schema:Currency
+rangeIncludes:UnitPriceSpecification
```

Comment:
nit: should this be Quantity?

Reply:
This is aligned with modeling in schema.org for currentExchangeRate. Do we want it to be Quantity instead?

Reply:
It isn't quite a price, and UnitPriceSpecification doesn't reference Quantity which we would need.

Reply:
sg, changing to quantity!